grand_challenge_forge/partials/docker-bash-scripts/do_build.sh.j2

```diff
@@ -6,12 +6,7 @@ set -e
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 DOCKER_IMAGE_TAG="{{ image_tag }}"
 
-
-# Check if an argument is provided
-if [ "$#" -eq 1 ]; then
-    DOCKER_IMAGE_TAG="$1"
-fi
-
```

Comment:
Why was the option to override the image tag removed?

Reply:
tbh, it sort of sneaked in. This made debugging the error I ran into so a tad more difficult so I removed it temporarily. Haven't return it because at the same time I had the plan to eventually use it to select an interface to run for local testing.

Do you think it was heavily used?

Reply:
Ok! No idea if it was used. It's probably fine to remove. I just saw it and didn't understand the connection to the rest of the PR and wanted to make sure I'm not missing something.